pyccel/ast/core.py

```diff
@@ -2799,7 +2722,7 @@ def __init__(self, source, target = None, ignore_at_print = False):
         self._source = source
         self._target = []
         self._ignore_at_print = ignore_at_print
-        if isinstance(target, (str, Symbol, DottedName, AsName)):
+        if isinstance(target, (str, PyccelSymbol, DottedName, AsName)):
```

Comment:
No need to add `PyccelSymbol` among the classes

Reply:
7bbafd8